doc/book/getting-started/database-and-models.md

```diff
@@ -217,10 +217,10 @@ class Module implements ConfigProviderInterface
         return [
             'factories' => [
                 Model\AlbumTable::class =>  function($container) {
-                    $tableGateway = $container->get(Model\AlbumTableGateway::class);
+                    $tableGateway = $container->get('Model\AlbumTableGateway');
```

Comment:
The original was correct. The file declares the namespace `Album`, so the original was being resolved relative to that.